src/components/ar-viewer.tsx

```diff
@@ -0,0 +1,182 @@
+import { useState, useEffect } from "react";
+import * as THREE from "three";
+import { GLTFLoader } from "three/addons/loaders/GLTFLoader.js";
+
+function detectARCapability() {
```

Comment:
please try not to show the AR button on desktop since they dont support AR (tested on macOS with chrome)